### PR TITLE
Admin signup

### PR DESCRIPTION
**Pull request by sturmer-eha**
openam authentication module added

### CODE REVIEW

**Comment by arnoldbm** [comment]
Can we put the password in an environment variable or meteor settings please, don't want to check that into git, and maybe not hardcode the 'openam.eha.io'/<port>, also put in settings please.

File: app/packages/openam/signup.coffee [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Template.signup.events
+  'submit form': (event, instance) ->
+    event.preventDefault()
+
+    Meteor.call("registerNewUser", "yursky555@blurg.com", "P@ssw0rd")
```
Comment (arnoldbm) [inline app/packages/openam/signup.coffee]: Is this a test method you accidentally left in?

**Comment by sturmer-eha** [comment]
I've used the hardcoded password just for testing, now it all comes from the form input. I'll make sure to clean up the forgerock accounts I've created during the development.

**Comment by sturmer-eha** [comment]
If there are no objections, I'd like to merge this branch into master.

File: app/client/routes/main.coffee [hunk 2]
```diff
@@ -1,50 +1,74 @@
...
+    unless Meteor.loggingIn() or Meteor.userId()
+      route = FlowRouter.current()
+      # unless route.route.name is 'login'
+        # Session.set 'redirectAfterLogin', route.path
+      FlowRouter.go 'login'
+  ]
+
+
+adminRoutes = loggedIn.group
+  prefix: '/admin'
```
Comment (jgoley) [inline app/client/routes/main.coffee]: I wonder if we need to prefix the routes with 'admin'. Will we ever need routes associated with non-admin users?

**Pull request merged**